packages/components/src/List/ListComposition/TextFilter/__snapshots__/TextFilter.component.test.js.snap

```diff
@@ -6,131 +6,107 @@ exports[`TextFilter should render text filter component 1`] = `
   initialDocked={true}
 >
   <withI18nextTranslation(FilterBar)
```

Comment:
Same principle as before

As in a previous PR, this is a simple change that is displayed in an unreadable manner :D 
The i18n HOC mock returns the original component with a modified name `withI18nextTranslation(name)`

Before
```
<withI18nextTranslation(FilterBar)>
    <FilterBar>
        ....
    </FilterBar>
</withI18nextTranslation(FilterBar)>
```

After 
```
<withI18nextTranslation(FilterBar)>
        ....
</withI18nextTranslation(FilterBar)>
```